Sources/SwifterSwift/SwiftStdlib/RawRepresentableExtensions.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  RawRepresentableExtensions.swift
+//  SwifterSwift
+//
+//  Created by 方林威 on 2020/7/1.
+//  Copyright © 2020 SwifterSwift
+//
+
+// MARK: - Initializers
+public extension RawRepresentable {
+    
+    /// SwifterSwift: Initializes an  enumeration type. If `rawValue` was malformed, returns `nil`.
+    /// - Parameters:
+    ///   - rawValue: Initialize the value of the enum type..
+    init?(rawValue: RawValue?) {
+        guard let value = rawValue else {
+            return nil
+        }
```

Comment:
Put these on 1 line
```suggestion
        guard let value = rawValue else { return nil }
```

Reply:
Hahaha, I usually write it this way, because I see some code in swifter is line break.

Reply:
I agree with this way of writing